Fix Shopify masking (#5282)
ethyca/fides: CHANGELOG.md

- Added performance mark timings to debug logs for fides.js [#5245](https://github.com/ethyca/fides/pull/5245)
- Initialized Ant Design and Tailwindcss in Admin-UI to prepare for Design System migration [#5308](https://github.com/ethyca/fides/pull/5308)

### Fixed
- Fix wording in tooltip for Yotpo Reviews [#5274](https://github.com/ethyca/fides/pull/5274)

> - Updated Shopify Datamap in order to ignore country and province values that are required on the erasure API

Reviewer: ```suggestion
- Updated Shopify dataset in order to flag country, province, and other location values as read-only
```
